arangod/RestHandler/RestWalAccessHandler.cpp

```diff
@@ -163,7 +163,7 @@ RestStatus RestWalAccessHandler::execute() {
     return RestStatus::DONE;
   }
 
-  if (ExecContext::CURRENT == nullptr || !ExecContext::CURRENT->isAdminUser()) {
+  if (!_context.isAdminUser()) {
```

Comment:
this seems to change the behavior in case authentication is turned off.
Previously it seems to have returned "forbidden" in that case, and now it will return success.
However, I think the previous behavior was a bug.

Reply:
ExecContext::CURRENT was never null in rest handlers